S/SCIP/build_tarballs.jl

```diff
@@ -7,19 +7,35 @@ version = v"800.0.401"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://scipopt.org/download/release/scipoptsuite-8.0.4.tgz", "be4f978be7f8f97371ddcdac7a60af69a4fea5f975090fe35f1ae4308db692d3"),
+    ArchiveSource(
+        "https://scipopt.org/download/release/scipoptsuite-8.0.4.tgz",
+        "be4f978be7f8f97371ddcdac7a60af69a4fea5f975090fe35f1ae4308db692d3"
+    ),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 # needed for now
 # clock_gettime requires linking to librt -lrt with old glibc
 # remove when CMake accounts for this
+
+# Hint to find libstc++, required to link against C++ libs when using C compiler
+
+# libstc++ required for c++ libs when using C compiler
+if [[ "${nbits}" == 32 ]]; then
+    export LDFLAGS="-lrt -L${prefix}/lib -liconv -Wl,-rpath-link,/opt/${target}/${target}/lib"
+elif [[ "${target}" != *-apple-* ]]; then 
+    export LDFLAGS="-lrt -L${prefix}/lib -liconv -Wl,-rpath-link,/opt/${target}/${target}/lib64"
+else 
+    export LDFLAGS="-L${prefix}/lib -liconv -lrt"
+fi
+
 if [[ "${target}" == *86*-linux-gnu ]]; then
-   export LDFLAGS="-lrt"
-elif [[ "${target}" == *-mingw* ]]; then
-   # this is required to link to bliss on mingw
-   export LDFLAGS=-L${libdir}
+    if [[ "${nbits}" == 32 ]]; then
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib"
+    else
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib64"
```

Comment:
I think `-rpath-link` may be a false lead here. The error says:
```
... warning: libdl.so.2, needed by /workspace/destdir/lib/libipopt.so, not found ...
```
So my first question would be: is `libdl.so.2` actually around anywhere?  If not then the use `-rpath-link` is moot anyway.